explanation/security/security_suggestions.md

```diff
@@ -0,0 +1,177 @@
+   system software:
+
+   ```bash
+   sudo apt update && sudo apt upgrade
+   ```
+
+   You may want to use the `unattended-upgrade` package to fetch and install
+   security updates and bug fixes automatically:
+
+   ```
+   sudo apt install unattended-upgrades
+   ```
+
+   By default, `unattended-upgrade` runs daily, but this can be configured. See
+   the `unattended-upgrade`
+   [manual page](https://manpages.ubuntu.com/manpages/noble/en/man8/unattended-upgrades.8.html)
+   for details.
+
+1. **Manage your software**:
+
+   * Remove packages you don't need, to minimise the potential attack surface
+     you are exposing. See our article on
+     {ref}`Package management <package-management>` for more details.
+
+   * Avoid using third party repositories. If you need to download a package
+     from a third party repository, make sure you
+     {ref}`understand the risks and how to minimize them. <third-party-repository-usage>`.
+
+1. **Use the most up-to-date release** of Ubuntu. If you are on an older Ubuntu
+   release we have instructions on {ref}`how to upgrade <upgrade-your-release>`.
+ 
+1. **Use [Ubuntu Pro](https://ubuntu.com/pro)**, particularly if you are on an
+   older release of Ubuntu. Pro provides Enterprise-level security patching,
+   but is free for personal/business use on up to 5 machines. The most useful
+   Pro features for *any* Ubuntu Server are:
+
+   * [Expanded Security Maintenance (ESM)](https://ubuntu.com/security/esm)
+     which expands the Ubuntu LTS commitment on packages in Main from 5 years
+     to 10 years -- and now also covers packages in Ubuntu Universe.
+   
+   * [Livepatch](https://ubuntu.com/security/livepatch) applies kernel patches
+     for high and critical severity vulnerabilities while the system is running.
+     This avoids the need for an immediate reboot.
+
+   Most security patches can be fetched and applied automatically through the
+   `unattended-upgrade` package. For more details on using and monitoring
+   Ubuntu Pro via the command line, refer to the
+   [official documentation](https://canonical-ubuntu-pro-client.readthedocs-hosted.com/en/latest/).
+
+### Access Control
+
+1. **Use and enforce** the
+   [principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege):
+
+   * This means creating non-root user accounts with as few privileges as possible.
+   * Not using `sudo` (root access) except for administration tasks.
+   * For more details on basic access control, see our {ref}`guide on user management <user-management>`.
+
+### Network security
+
+1. **Use a firewall**. In Ubuntu, the uncomplicated firewall (`ufw`) tool is
+   used to configure firewalls. `ufw` is a wrapper around the `iptables` utility
+   (which experienced system admins may prefer to use directly). To get started
+   with `ufw`, check out our {ref}`firewalls` guide.
+
+1. **Use the Secure Shell (SSH)** protocol to secure remote access. In Ubuntu,
```

Comment:
Using ssh is fine, the art is doing it well.

We sadly lack content here on the pages to suggest proper key types and sizes, use passphrases, potentially suggest storing the private key securely as well as something like two factor like http://www.justgohome.co.uk/blog/2013/07/better-two-factor-ssh-authentication-on-ubuntu.html
If not there yet we at least should track that as an extension of the content.